src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -51,8 +51,9 @@ class AnonymousElectorDocument(
     @field:Size(max = 255)
     var sourceReference: String,
 
+    @field:NotNull
     @field:Size(max = 255)
-    var applicationReference: String?,
+    var applicationReference: String,
```

Comment:
This is mandatory in the generate endpoint within the swagger spec, but just to be sure we've checked that there aren't any null instances in prod.